src/EnergyPlus/HeatingCoils.cc

```diff
@@ -3320,6 +3344,73 @@ namespace HeatingCoils {
 
 	}
 
+	void
+	SetHeatingCoilData(
+		int const CoilNum, // Number of electric or gas heating Coil
+		bool & ErrorsFound, // Set to true if certain errors found
+		Optional_bool DesiccantRegenerationCoil, // Flag that this coil is used as regeneration air heating coil
```

Comment:
@Nigusse @rraustad  optional arguments are frowned upon now (see [Optional Arguments](https://github.com/NREL/EnergyPlus/wiki/Development-Tips-%26-Resources#optional-arguments).  preferred method is to have a default.  But I'm wondering why we even need this function.  If I'm following correctly - this function sets a flag on the heating coil object which is only used to trigger a special block of code in SizeHeatingCoil which does some special sizing steps if this coil is in a desiccant dehumidifier. But we don't have equivalent code to distinguish a coil that lives in a fan coil vs a reheat terminal unit.  It appears that we're introducing yet another method to communicate a special sizing condition.